pages/Identity/Overview.md

```diff
@@ -7,22 +7,319 @@ menu: Identity
 # Identity
 
 This specification is intended to cover the concept of identity within the protocol and how we represent it.
-It may include ideas for interfacing with external identity systems, methods to upgrade the identity system in the future and potential user authorization systems.
 
 ## Specification Status
 
 | Version | Status |
 ---------- | ---------
-| 0.1     | Proposed |
+| 0.1     | Draft |
 
 ## Purpose
 
-TBD
+1. Provide the interface for a DSNP identity
+1. Specify the delegation model and related interface
+1. Specify the ownership model and related interface
+1. Provide the list of EIPs that must be supported by a DSNP compatible identity 
 
 ## Assumptions
 
-TBD
+* Identity is on Ethereum for Betanet
+* Identity will need to be moved from Betanet to Mainnet
 
 ## Details
 
-TBD
+Identity is comprised of several parts:
+
+* Ownership
+* Delegation
+* Identification
+* Discovery
+
+Additional constraints were placed upon any solution:
+
+* Pseudo Anonymity: A consistent identifier that defaults to disconnected from real world identity 
+* Flexibility: An interface that can produce multiple solutions
+* Extensibility: A default implementation that can be updated over time as DSNP changes
+
+## Ownership
+
+On the blockchain ownership is determined by the ability to sign messages.
+To this end, ownership of an identity is also determined by the address of the creator.
+(Options for allowing a delegate to create an identity on behalf of someone is described in the [Identity Factory](/Identity/Factory).)
+
+Creation of any new identity MUST be authorized by the owner's address.
+The official [Identity Factory](/Identity/Factory) must be used for initial creation of a new identity.
+
+
+## Option 1: Single Owner
+
+Each identity contract may only have one owner, although delegations may provide for other methods of transferring ownership. 
+
+The contract MUST implement [EIP 173](https://eips.ethereum.org/EIPS/eip-173) which provides methods that confirm ownership and facilitate ownership transfer.
+
+## Option 2: Permissioned Owners
+
+Ownership is managed through using permissions.
+While at least one owner is required, additional public keys may be considered to have ownership and may remove or add other owners.
+
+A contract MAY implement [EIP 173](https://eips.ethereum.org/EIPS/eip-173) which provides standard methods that confirm ownership and facilitate ownership transfer.
+
+## Delegation
```

Comment:
The two options below were discussed in a meeting:

```
## Option 1: Single Permission

Delegates are assumed to all have the same permission, although that permission may be different than that of the owner.
Exact permissions are TBD, but MUST NOT include the ability to add additional delegates. 
```
and
```
### Option 2: With Permissions

Specific permissions are granted when the delegate is added.
```

While the list of potential specific permissions were limited, it was considered the better option due to flexibility and the minimal additional storage requirements. Additionally the need for being able to ask if `isAuthorizedTo` already needed, the increase in complexity was considered a worthwhile tradeoff, while still enabling a simple implementation.